.travis.yml

```diff
@@ -4,16 +4,13 @@ python:
   - "2.7"
   - "3.7"
 
-matrix:
-  allow_failures:
-    - python: "3.7"
-
 install:
-  - pip install pylint tox scapy
-  - pip install tornado==4.5.3
+  - pip install flake8 pylint scapy tornado tox
+  - pip install git+http://github.com/wifiphisher/roguehostapd.git
```

Comment:
Interesting, I wasn't aware this is possible. Me and @anakin1028 spent a lot of time troubleshooting with PyPi.